ui/components/app/assets/nfts/nft-empty-state/nft-empty-state.test.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { fireEvent, screen } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import { renderWithProvider } from '../../../../../../test/lib/render-helpers';
+import mockState from '../../../../../../test/data/mock-state.json';
+import { ThemeType } from '../../../../../../shared/constants/preferences';
+import { NftEmptyState } from './nft-empty-state';
+
+// Mock global platform
+const mockOpenTab = jest.fn();
+
+// @ts-expect-error mocking platform
+global.platform = {
+  openTab: mockOpenTab,
+};
+
+describe('NftEmptyState', () => {
+  const mockStore = configureMockStore([]);
+
+  const renderComponent = (props = {}, stateOverride = {}) => {
+    const store = mockStore({ ...mockState, ...stateOverride });
+    return renderWithProvider(<NftEmptyState {...props} />, store);
+  };
```

Comment:
render component renders `NftEmptyState with mock state and overrides to test light and dark mode images